face_track/face_atomic.py

```diff
@@ -34,6 +34,8 @@ def __init__(self):
 	def update_face_octomap(self, faceid, xx, yy, zz):
 		face="(map-ato \"faces\" (NumberNode \""+str(faceid)+"\") "+str(xx)+" "+str(yy)+" "+str(zz)+")\n"
 		netcat(self.hostname, self.port, face + "\n")
+		face = "(NumberNode \"" + str(faceid) + "\" (av 5 0 0))\n"
+		netcat(self.hostname, self.port, face + "\n")
```

Comment:
you can do this on one line, you don't need two lines.  Just like the other place where you made this change.